client/src/index.tsx

```diff
@@ -18,29 +18,31 @@ import {
 } from './pages';
 import Wizard from './wizard';
 import { PrivateRoute } from './components';
-import { AuthProvider, TeamsProvider } from './contexts';
+import { AuthProvider, TeamsProvider, InterestsProvider } from './contexts';
 
 import './styles/styles.scss';
 
 ReactDOM.render(
   <React.StrictMode>
     <AuthProvider>
-      <TeamsProvider>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <Redirect to="/login" />
-            </Route>
-            <PrivateRoute exact path="/pitches" component={PitchDoc} />
-            <PrivateRoute exact path="/resources" component={Resources} />
-            <PrivateRoute exact path="/profile/:userId" component={Profile} />
-            <PrivateRoute exact path="/users" component={Directory} />
-            <PrivateRoute exact path="/login" component={Login} />
-            <PrivateRoute exact path="/join" component={Wizard} />
-            <PrivateRoute exact path="*" component={NotFound} />
-          </Switch>
-        </Router>
-      </TeamsProvider>
+      <InterestsProvider>
+        <TeamsProvider>
+          <Router>
+            <Switch>
+              <Route exact path="/">
+                <Redirect to="/login" />
+              </Route>
+              <PrivateRoute exact path="/pitches" component={PitchDoc} />
+              <PrivateRoute exact path="/resources" component={Resources} />
+              <PrivateRoute exact path="/profile/:userId" component={Profile} />
+              <PrivateRoute exact path="/users" component={Directory} />
+              <PrivateRoute exact path="/login" component={Login} />
+              <PrivateRoute exact path="/join" component={Wizard} />
+              <PrivateRoute exact path="*" component={NotFound} />
+            </Switch>
+          </Router>
+        </TeamsProvider>
+      </InterestsProvider>
     </AuthProvider>
```

Comment:
Should we create a component to wrap all the providers so that we only have a single level of indentation for them all here?

Reply:
Not a bad idea. I can do it